UPGRADING.md

```diff
@@ -1,5 +1,12 @@
 # Upgrade guidelines
 
+## 0.87.0 to 0.88.0
+
+The `networkExplorer` option is deprecated now. Please use the upstream OpenTelemetry eBPF Helm chart to collect
```

Comment:
Should we add more instructions about how to switch? Is it recommended for users to first disable the nw explorer from Splunk distro release and then install the new release? I assume we might not want both the running simultaneously.

Reply:
Good point. updated 👍